velociraptor/catalogue/translator.py

```diff
@@ -8,6 +8,1396 @@
 from velociraptor.units import VelociraptorUnits
 
 
+def VR_to_SOAP(particle_property_name: str):
+    """
```

Comment:
Oh my goodness, I hate it. Though I understand why it is needed for now. We need to make sure this remains temporary.

Reply:
I totally agree.